software.md

```diff
@@ -0,0 +1,31 @@
+# FOSS United Community Software Policy
+This policy is devised to guide, but not mandate, software choices within the 
+FOSS United Community.
+
+- Define what we (the FOSS United Community) wish to do. Ask ourselves 
+whether what we wish to do will help in full-fulling the community's goals.
+- Define your requirements for the software solution clearly. Clearly 
+differentiate between needs and wants. Needs are features that are crucial to 
+performing the task at hand and wants are niceties, things we can do without.
+- Look for FOSS solutions that are capable of meeting the needs at a level 
+that's acceptable to the community. Focus should be on the needs.
+- Look at the cost in terms of pricing, effort and time that the community needs 
+to spend to be able to use solution and constantly look for ways we can lower 
+such barriers and consider supporting the project if that helps.
+- When a solution is available in the FOSS ecosystem use it.
+- When such a solution is not available in the ecosystem assuming that this is 
+the only thing that prevents us from doing something that will benefit the cause 
+of FOSS United then look for proprietary solutions.
+- Assess the costs of each solution in terms of privacy, freedom, pricing, effort 
+and time that the community needs to spend to be able to use software and then 
+make a choice as a community.
+- All choices needs to be made in a democratic fashion. Enough time and effort 
+should be spent to form a collective decision as a community rather than make 
+decisions based on the opinions of a few people that happened to be online 
+when the issue was being discussed.
+
+Software choices are not meant to be forced onto individual people volunteering 
+their time and effort for FOSS United. For example, a graphic designer producing
+an visual for the the community would not be expected to examine their choice of 
+tools. Most importantly, any output should be available in an open format, usable
+by a variety of tools.
```

Comment:
Again, the scope of this policy is not very clear to me. Does it only apply to adopting new software for the community usage? or to every poster that we make and every line of code that we write?

Reply:
The purpose of that line is to state that it only applies to adopting new software for the community usage and not decisions regarding individual choices.

Sticking with the BBB example, that instance is going to be used by individual community members so it is important that the community gets a say in that (because we're a FOSS Community) but when making a poster only the person working on it needs to use the software that's used to make the poster. So the choice regarding that software is a personal choice and is outside the scope of this policy.

However if we ever wish to reuse the poster's source file (say a .ai file) to create another poster we'd have to use illustrator too. So it's important that the source file is available in an open format such as svg so that anyone can work on it from any software of their choice.

Hope this clears it up.